sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -660,4 +660,34 @@ public void createWithAutoscale() throws ClassNotFoundException {
         assertEquals(Integer.parseInt(TestConstants.AUTOSCALE_MAX_THROUGHPUT),
             throughput.getProperties().getAutoscaleMaxThroughput());
     }
+
+    @Test
+    public void createDatabaseWithThroughput() throws ClassNotFoundException {
+        final String dbName = TestConstants.DB_NAME + "-other";
+        deleteDatabaseIfExists(dbName);
+
+        Integer expectedRequestUnits = 700;
```

Comment:
Why 700 here ?
Where are we setting this?

Reply:
No particular reason for 700, the test is simply intended to verify that if the database throughput is provided, the database is created with that specific throughput.  When defining the database throughput, some value for the request units needs to be provided - i just pulled 700 out of the air but it can be whatever.  If you'd prefer another value, let me know and I can put that in.

Reply:
in answer to your question, it's used just below, when the CosmosConfig is created...
```
final CosmosConfig config = CosmosConfig.builder()
            .enableDatabaseThroughput(false, expectedRequestUnits)
            .build();
            ```
 and then again when comparing the value of the created database actual throughput...
 ```
 assertEquals(expectedRequestUnits, response.getProperties().getManualThroughput());
 ```

Reply:
No need to change the value, I was just curious about 700 :) 
The test looks good to me.